### PR TITLE
add AP to Tensorboard

### PR DESCRIPTION
**Pull request by brunoeducsantos**
Hi,
This PR is meant to give in verbose option to add AP to tensorboard.

Regards,
Bruno

### CODE REVIEW

File: keras_retinanet/utils/compute_overlap.c [hunk 1]
```diff
@@ -0,0 +1,8701 @@
+/* Generated by Cython 0.29.15 */
```
Comment (hgaiser) [inline keras_retinanet/utils/compute_overlap.c:1]: I don't think you intended to commit this file?

**Comment by hgaiser** [comment]
What does this do exactly? Don't we already write to tensorboard? Or does this write the per class AP scores?

**Comment by brunoeducsantos** [comment]
Hi @hgaiser ,
I want to write per class the AP scores on tensorboard.

**Comment by hgaiser** [comment]
Sorry for the delay, but thanks for the PR ^^

**Pull request merged**